components/director/internal/info/handler.go

```diff
@@ -7,21 +7,42 @@ import (
 	"github.com/kyma-incubator/compass/components/director/pkg/httputils"
 )
 
+// Implemented ORD Version
+const ordVersion = "1.1.0"
```

Comment:
It is probably better to move this constant to the open_resource_discovery package and export it.

On one hand, it is domain-specific and fits better, on the other hand, if you add it here on each version increasing a review from team-raptor will be needed.

Reply:
done